src/pretix/plugins/sendmail/templates/pretixplugins/sendmail/send_form.html

```diff
@@ -10,12 +10,32 @@
             {% bootstrap_field form.sendto layout='horizontal' %}
             {% bootstrap_field form.subject layout='horizontal' %}
             {% bootstrap_field form.message layout='horizontal' %}
+            {% if request.method == "POST" %}
+            <fieldset>
+            <legend>{% trans "E-mail preview" %}</legend>
+
+                <div class="panel-body">
+                    {% for k,v in output.items %}
+                    <p>
+                        <strong>For locale: </strong>{{ k }}
```

Comment:
We need to have this string translatable as well. This could be achieved like this:
```
<strong>{% blocktrans %}Preview for locale: {{ k }}{% endblocktrans %}</strong>
```
This will show something like ``Preview for locale: de``. It would obviously be even better to show ``Preview for locale: German``, but I don't see an easy way to achieve this (we'd need to somehow read this data from ``settings.py``), I suggest we keep it with ``de`` for now.